src/pycountry/db.py

```diff
@@ -194,7 +191,7 @@ def get(self, **kw: Optional[str]) -> Optional[Any]:
             return default
 
     @lazy_load
-    def lookup(self, value: str) -> Type:
+    def lookup(self, value: str) -> D:
```

Comment:
This should have been `Any` not `Type` since this is not returning a factory, but is returning an instance of `Type`. Now that this is a generic class it returns the generic type.